src/lib/web-component/filter-option/filter-option-element.js

```diff
@@ -26,6 +28,10 @@ class FilterOption extends HTMLElement {
 
   updateFilterCheckboxes(logs) {
     for (const log of logs) {
+      if (this.filterKey === 'name' && log.type === 'content_script') {
```

Comment:
This doesn't belong here. You should control this at the input, e.g. by using `logs.filter(log => ...)` to exclude some entries.